includes/core-changes/windowsforms/cellformatting-event-not-raised.md

```diff
@@ -0,0 +1,33 @@
+### CellFormatting event not raised if tooltip is shown
+
+To meet accessibility standards, a <xref:System.Windows.Forms.DataGridView> now shows a cell's text and errors as tooltips when navigated by both mouse and keyboard. If a tooltip is shown, the <xref:System.Windows.Forms.DataGridView.CellFormatting?displayProperty=nameWithType> event is not raised.
```

Comment:
Is "when navigated" generally accepted language for this?  It doesn't appear in the [style guide list of terms for mouse interactions](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29013) or in the [Describing interactions with UI article](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=26472).   Is it referring to "when hovering over" with the mouse or "when selected" by the keyboard? My understanding of "navigating" to an element with a mouse would include selecting it, which is different from "hovering" over it. 

Should "both ... and ..." be "either ... or ..."?

Would it be accurate to change the last sentence to something like "If a tooltip is shown **for a cell**, the DataGridView.CellFormatting event is not raised **for other cells**."?

The "recommended action" section is brief and clear, could this summary be replaced by something like "The CellFormatting event is not raised while the DataGridView is in edit mode." and leave the details for the description section?